<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicConnectTransferWidget</class>
 <widget class="QWidget" name="MusicConnectTransferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>传文件到移动设备</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>897</width>
     <height>542</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>传文件到移动设备</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>879</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>897</width>
     <height>517</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:white;}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QWidget" name="playListLayoutWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>150</y>
      <width>120</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playListLayout">
     <property name="spacing">
      <number>2</number>
     </property>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>85</y>
      <width>897</width>
      <height>1</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>1</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(200, 200, 200, 150)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::HLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>85</y>
      <width>1</width>
      <height>432</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(200, 200, 200, 150)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::VLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>0</y>
      <width>1</width>
      <height>517</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(200, 200, 200, 150)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::VLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>95</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchLineLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tiny/btn_search_main_hover);</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="MusicConnectTransferTableWidget" name="listTableWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>361</width>
      <height>330</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="allSelectedcheckBox">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>95</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="transferUSBButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>467</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>transfer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>5</y>
      <width>85</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/toolSets/lb_mobile_computer</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>5</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/toolSets/lb_mobile_usb</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:fold;font-size:20px;color:rgb(100,100,100)</string>
    </property>
    <property name="text">
     <string>我的电脑</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceInfoLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:fold;font-size:20px;color:rgb(100,100,100)</string>
    </property>
   </widget>
   <widget class="QLabel" name="songCountLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>115</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c8c8c8;&quot;&gt;共 &lt;/span&gt;&lt;span style=&quot; color:#646464;&quot;&gt;%1 &lt;/span&gt;&lt;span style=&quot; color:#c8c8c8;&quot;&gt;首歌曲&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>455</y>
      <width>712</width>
      <height>1</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>1</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(200, 200, 200, 150)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::HLine</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="selectCountLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#c8c8c8;&quot;&gt;已选 &lt;/span&gt;&lt;span style=&quot; color:#646464;&quot;&gt;%1 &lt;/span&gt;&lt;span style=&quot; color:#c8c8c8;&quot;&gt;首歌 | 共 &lt;/span&gt;&lt;span style=&quot; color:#646464;&quot;&gt;%2M&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MusicConnectTransferCompleteTableWidget" name="completeTableWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>333</width>
      <height>330</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicConnectTransferTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicconnecttransfertablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MusicConnectTransferCompleteTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicconnecttransfertablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../TTKUi/TTKModule.qrc"/>
 </resources>
 <connections/>
</ui>
